python/core/auto_generated/symbology/qgsmarkersymbollayer.sip.in

```diff
@@ -103,6 +103,15 @@ Encodes a shape to its string representation.
     virtual QRectF bounds( QPointF point, QgsSymbolRenderContext &context );
 
 
+    QPainterPath path();
+%Docstring
+Returns the symbol layer path as a QPainterPath
```

Comment:
Needs some extra explanation here. E.g.

- what coordinate system is the path in? Painter units? Layer units? 
- can we only call this method safely between startRender/stopRender calls, like several other methods in this class?

Reply:
also no longer needed